### PR TITLE
Creates contributing rules

### PR DESCRIPTION
**Pull request by waldemarnt**
I'm working to document our patterns about how to contribute and write in our documentation, can you folks help me on that?

### CODE REVIEW

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -1,16 +1,71 @@
-# Before You Submit an Issue
+# Contributing to AGCO Documentation
 
-- Are you using Windows? We unfortunately don't support Windows. You could try using Docker, as outlined in the `README`.
-- Is your version of Slate out of date? We have [upgrade instructions](https://github.com/tripit/slate/wiki/Updating-Slate) in the wiki. You could also try seeing if your problem is reproducible on the latest version of Slate.
+The following topics are a set of guidelines for contributing. These guidelines helps
+to have a pattern that everyone can read and start contributing.
 
-If you answered "no" to each of the questions above, feel free to submit an issue! It's also helpful if you include a code example of your problem (if applicable), we can't help you if you just say "Slate stopped loading for me once I added my documentation" without telling us the problematic documentation.
+#Style guide
```
Comment (jeffhsta) [inline CONTRIBUTING.md]: This section is blank.

File: CONTRIBUTING.md [hunk 2]
```diff
@@ -1,16 +1,71 @@
...
 
-# Before You Submit a Pull Request
+#General guidelines
```
Comment (jeffhsta) [inline CONTRIBUTING.md]: This section is blank.

File: CONTRIBUTING.md [hunk 3]
```diff
@@ -1,16 +1,71 @@
...
 
-Thanks for contributing to Slate! A couple of quick guidelines for submitting pull requests:
+##Documenting APIs
 
-- **Please point your pull requests at the `dev` branch.** We don't accept pull requests to `master`.
-- Please make sure your contributions work in the most recent version of Chrome, Firefox, and IE.
-- If you're implementing a new feature, even if it's relatively small, it's nice to open an issue before you start so that others know what you're working on and can help make sure you're on the right track.
+APIs must be placed on the left panel using H1 with `#` .
```
Comment (jeffhsta) [inline CONTRIBUTING.md]: s/APIs/High leve topic/
I also would add the the H2 or `##` will also appear on the left panel as a subtopic.

File: CONTRIBUTING.md [hunk 4]
```diff
@@ -1,16 +1,71 @@
...
\ No newline at end of file
+##Documenting resources
+
+This guideline applies for every resource that should be documented. These resources should be placed
+inside of their application headers using H2 with `##` as the example below:
+
+#Telemetry API
+##Brands
+
+Every resource should contains their *request headers* if have some different from the default,
```
Comment (lucaslago) [inline CONTRIBUTING.md]: s/contains/contain

Comment (jeffhsta) [inline CONTRIBUTING.md]: When you say *resource* are you referring to a *REST resource*?

File: CONTRIBUTING.md [hunk 5]
```diff
@@ -1,16 +1,71 @@
...
+*request parameters* if have some different from the default,
```
Comment (lucaslago) [inline CONTRIBUTING.md]: duplicated line?

File: CONTRIBUTING.md (hunk 4 above)
Comment (jeffhsta) [inline CONTRIBUTING.md]: s/contains/contain/
I am not sure but I think its make more sense

File: CONTRIBUTING.md [hunk 6]
```diff
@@ -1,16 +1,71 @@
...
+*http request example* and *http response example* using the [HTTP request/response pattern](#documenting-request-response).
+
+##Documenting request/response
+Every request and response should be documented using the [HTTP standard provided by slate](https://github.com/lord/slate/wiki/Customizing-the-Language-Tabs).
+
+### request
```
Comment (lucaslago) [inline CONTRIBUTING.md]: s/request/HTTP Request

File: CONTRIBUTING.md [hunk 7]
```diff
@@ -1,16 +1,71 @@
...
+The request also must have *Host Header* and if the resource is private the example must contains the *Authorization Header* as well,
+as the example below:
+
+```http
+GET /models HTTP/1.1
+Authorization: <A Valid Token>
+Host: agco-fuse-trackers-sandbox.herokuapp.com
+```
+
+### response
```
Comment (lucaslago) [inline CONTRIBUTING.md]: s/response/ HTTP response

File: CONTRIBUTING.md [hunk 8]
```diff
@@ -1,16 +1,71 @@
...
+APIs must be placed on the left panel using H1 with `#` .
 
-Thanks again! Happy coding.
\ No newline at end of file
+##Documenting resources
+
+This guideline applies for every resource that should be documented. These resources should be placed
+inside of their application headers using H2 with `##` as the example below:
+
+#Telemetry API
```
Comment (lucaslago) [inline CONTRIBUTING.md]: What do you think of refactoring this example as code? e.g:
```markdown
## Brands
  Every resource should contains...
```

File: CONTRIBUTING.md [hunk 9]
```diff
@@ -1,16 +1,71 @@
...
+as the example below:
+
+```http
+GET /models HTTP/1.1
+Authorization: <A Valid Token>
+Host: agco-fuse-trackers-sandbox.herokuapp.com
+```
+
+### response
+The response must contains the *status code* and the *Content-Type Header*.
```
Comment (lucaslago) [inline CONTRIBUTING.md]: s/contains/contain

File: CONTRIBUTING.md [hunk 10]
```diff
@@ -1,16 +1,71 @@
...
+If the response contains a body, this body should be placed below the response headers and status code like the example:
```
Comment (lucaslago) [inline CONTRIBUTING.md]: s/this body/it

File: CONTRIBUTING.md (hunk 1 above)
Comment (waldemarnt) [inline CONTRIBUTING.md]: @jeffhsta this will be incremented as we work.

File: CONTRIBUTING.md (hunk 2 above)
Comment (waldemarnt) [inline CONTRIBUTING.md]: @jeffhsta  this will be incremented as we work.

**Comment by lucaslago** [comment]
👍

**Comment by jeffhsta** [comment]
:+1:

**Pull request merged**